BedrockServer.cpp

```diff
@@ -826,7 +826,7 @@ void BedrockServer::worker(SData& args,
                                 uint64_t preLockTime = STimeNow();
                                 shared_lock<decltype(server._syncThreadCommitMutex)> lock1(server._syncThreadCommitMutex);
                                 SINFO("_syncThreadCommitMutex acquired in worker in " << fixed << setprecision(2)
-                                      << ((STimeNow() - preLockTime)/1000.0) << "ms.");
+                                      << ((STimeNow() - preLockTime)/STIME_US_PER_MS) << "ms.");
```

Comment:
I think you misunderstood this line in particular (it logs decimal milliseconds already).

Reply:
I was just changing the extant 1000 to the constant "while I was there" for consistency. Now it's consistently annoying. :)